package.json

```diff
@@ -15,8 +15,8 @@
     "load-addresses": "yarn workspace @uma/core load-addresses",
     "create-release": "lerna version --no-git-tag-version --conventional-commits --no-push",
     "publish-release": "./scripts/run_release.sh",
-    "qbuild": "yarn workspace @uma/core build && yarn lerna run build --stream --ignore '*/*dapp*'",
-    "build": "yarn workspace @uma/core build && yarn lerna run build --stream",
+    "qbuild": "yarn workspace @uma/common build && yarn workspace @uma/core build && yarn lerna run build --stream --ignore '*/*dapp*' --ignore '@uma/common' --ignore '@uma/core'",
```

Comment:
why do you need this diffrent syntax on these builds now?

Reply:
Two points I noticed:
- We were re-running the core build because we were running it as a pre-step and then it was also being run as part of the second step, so I just removed the redundancy. Unrelated to the typechain change.
- Common is technically a dependency of core. The core `.ts` files now use types from common, so it needs to be built before core, so I needed to specifically run it first to preempt the core build. Later, I would like to figure out how to force lerna to figure out the right order automatically, but I think it currently doesn't work because we have some circular deps.

Reply:
Gonna go ahead and merge since this seems pretty minor, but happy to refactor in a follow up if you have a different suggestion!